test/shm_size.bats

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+  setup_test
+}
+
+function teardown() {
+  cleanup_test
+}
+
+@test "check /dev/shm is changed" {
+  # Start it with the environment variable set, don't use replace_config
+  CONTAINER_ENABLE_CUSTOM_SHM_SIZE=true start_crio
+
+  # Run base container to ensure it creates at all
+  run crictl runp <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"')
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  # Run multi-container pod to ensure that they all work together
+  pod_id="$output"
+  run crictl create "$pod_id" "$TESTDATA"/container_sleep.json <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"')
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  # Set container id
+  ctr_id="$output"
+  run crictl start "$ctr_id"
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  # No clue why running this twice makes it work
+  run crictl exec --sync "$ctr_id" df /dev/shm
+  echo "$output"
+  run crictl exec --sync "$ctr_id" df /dev/shm | grep '/dev/shm' | awk '{print $2}'
```

Comment:
I don't believe we need awk, below we're just checking if 16384 is a substring

Reply:
Testing some variations, it seems to hate the pipes in the end of the crictl exec, this version seems to work:

```
run crictl exec --sync "$ctr_id" df
  [[ $(echo "$output" | grep '/dev/shm') == *"16384"* ]]
```